compiler/test/dotty/tools/dotc/MissingCoreLibTests.scala

```diff
@@ -2,32 +2,21 @@ package dotty
 package tools
 package dotc
 
-import org.junit.{AfterClass, Test}
-import vulpix._
+import org.junit.Test
+import org.junit.Assert._
 
-import scala.concurrent.duration._
+import vulpix.TestConfiguration.mkClassPath
 
-class MissingCoreLibTests extends ParallelTesting {
-  import MissingCoreLibTests._
-  import TestConfiguration._
+class MissingCoreLibTests {
 
-  // Test suite configuration --------------------------------------------------
-
-  def maxDuration = 30.seconds
-  def numberOfSlaves = 5
-  def safeMode = Properties.testsSafeMode
-  def isInteractive = SummaryReport.isInteractive
-  def testFilter = Properties.testsFilter
-
-  @Test def missingDottyLib: Unit = {
+  @Test def missingDottyLib2: Unit = {
     val classPath = mkClassPath(Jars.dottyCompiler :: Jars.dottyInterfaces :: Jars.dottyExtras) // missing Jars.dottyLib
-    val options = noCheckOptions ++ checkOptions ++ yCheckOptions
-    compileFile("../tests/neg/nolib/Foo.scala", TestFlags(classPath, options)).checkExpectedErrors()
+    val source = "../tests/neg/nolib/Foo.scala"
+    val options = Array("-classpath", classPath, source)
+    val reporter = Main.process(options)
```

Comment:
You are not handling the case where the compiler crashes, vulpix did that. A `try/catch` and `assertTrue(false, ex.stackTrace)` a good enough replacement.

Reply:
My understanding is that if something in the test crashes, the tes fail

Reply:
Right